### ISSUE TITLE
Adding datapack to pre-datapack worlds causes crash

### ISSUE THREAD

**Issue by TrillionDeTurtle** [onnowhere/examples#1]
is there a way to apply this to my long term survival world and if so please could I have detailed instructions on how to do it

**Comment by Picoseconds** [comment]
Which datapack are you talking about?

**Comment by Koodaamo-MareDev** [comment]
I think they mean that they have a world that was generated using the old build height which is incompatible with the new build height. I don't think there's currently any way to do it and if it crashes I think it's up to Mojang to fix it. Feel free to correct me if I'm wrong.

**Comment by onnowhere** [comment]
The pack examples provided won't generate any land with exception of the 512 height one. You'll have to wait until someone releases a version that supports all the vanilla land generation while also supporting those ones as the other two cause issues with the world being unable to load blocks. Not sure how the 512 height one impacts an existing world but I will need to test it.

**Comment by onnowhere** [comment]
Yeah, MarmoriPelaa is right, just tried it and the world crashed. Sorry about that, but hopefully they will look into it sometime, though can't guarantee that it would be fixed the way we hope as they could just disable the datapack when it's found to be incompatible or crash-prone. We'll just have to see what methods can be possible as time goes on.
